myExpenses/src/main/java/org/totschnig/myexpenses/fragment/TransactionList.kt

```diff
@@ -48,6 +55,36 @@ class TransactionList : BaseTransactionList() {
         }
     }
 
+    override fun resetTransactionSum() {
+        selectedTransactionSum = 0
+    }
+
+    override fun setTitle(mode: ActionMode, lv: AbsListView) {
+        val count = lv.checkedItemCount
+        mode.title = setColor(count.toString() + " " + currencyFormatter.convAmount(selectedTransactionSum, mAccount.currencyUnit), count + 1)
```

Comment:
It should be simpler to call setColor only with the amount. Then the count argument is not necessary. The result can be concatenated with TextUtils.concat: